.github/workflows/tests.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8]
```

Comment:
May as well add 3.9 as I assume we will one day need to migrate again.

Reply:
I now see that 3.9 is not supported by EB. No problem.

Reply:
I'm open to adding 3.9 and 3.10 to the test action to check if they would pass and then keep them in if they do. If the tests need work, then let's push it off to another PR.

Reply:
Up to you, I like testing lots of versions but no EB support is a good reason not to include it.

Reply:
Eh I'll put it another PR let's just merge this in